proto/pdpb.proto

```diff
@@ -377,6 +378,9 @@ message BatchScanRegionsRequest {
 
     repeated KeyRange ranges = 3; // the given ranges must be in order.
     int32 limit = 4; // limit the total number of regions to scan.
+    // If output_must_contain_all_key_range is true, the output must contain all key ranges in the request.
+    // If the output does not contain all key ranges, the request is considered failed and returns an error(REGIONS_NOT_CONTAIN_ALL_KEY_RANGE).
+    bool output_must_contain_all_key_range = 5;
```

Comment:
Can we make it shorter?

Reply:
Do you have some better suggestions?😂

Reply:
contain_all_key_range?

Reply:
What do you think which is better? @lance6716 @you06

Reply:
With the comment explaination, `contain_all_key_range` is clear enough to me.

Reply:
@you06 updated in https://github.com/pingcap/kvproto/pull/1252/commits/49bf52b9e60090df9f0eb3040a2ce7ae84edbe4f